File: AD-HOC/1765/main.dart
import 'dart:io';

void main() {
  int t;
  while ((t = int.parse(stdin.readLineSync()!)) > 0) {
    for (int i = 0; i < t; i++) {
      var [q, a, b] = stdin.readLineSync()!.split(' ').map(double.parse).toList();
      print('Size #${i + 1}:');
      print('Ice Cream Used: ${(q * (a + b) * 2.5).toStringAsFixed(2)} cm2');
    }
    print('');
  }
}

File: AD-HOC/1765/Main.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in).useLocale(Locale.ENGLISH);
        int t;
        while ((t = scanner.nextInt()) > 0) {
            for (int i = 0; i < t; i++) {
                int q = scanner.nextInt();
                double a = scanner.nextDouble();
                double b = scanner.nextDouble();
                System.out.printf("Size #%d:\n", i + 1);
                System.out.printf("Ice Cream Used: %.2f cm2\n", q * (a + b) * 2.5);
            }
            System.out.println();
        }
        scanner.close();
    }
}
